modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/extensions/aspects/ProcessLifeCycle.java

```diff
@@ -0,0 +1,883 @@
+/*
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7